types/koa-router/koa-router-tests.ts

```diff
@@ -78,7 +78,7 @@ app.use(async (ctx: Koa.ParameterizedContext<MyState, MyContext>, next) => {
 
 const router3 = new Router();
 router3.get('/', (ctx) => {
-    ctx.foo = "bar";
```

Comment:
The rest of the code only operates on `ctx.state.foo` - this seems to be a bug found by the more strict typing.